CHANGELOG.md

```diff
@@ -1,3 +1,67 @@
+# 1.7.0-alpha02 (July 2024)
+_Changes since 1.7.0-alpha01_
```

Comment:
[`MD022` - Headings should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md022.md)